AppboyProject/ios/Podfile

```diff
@@ -1,5 +1,11 @@
-
-target 'AppboyProject' do 
+target 'AppboyProject' do
   platform :ios, '8.0'
+
+  # if you want to integrate through Cocoapods instead of manual linking use this:
+  # it already comes with a dependency on `Appboy-iOS-SDK` so you're all set
+  # pod 'react-native-appboy-sdk', :path => '../node_modules/react-native-appboy-sdk'
```

Comment:
Per your https://github.com/adjust/react_native_sdk/pull/14#issuecomment-323934886, could you also include `React` and `Yoga` here for sample integration purposes?